SPECS/jitterentropy/jitterentropy.spec

```diff
@@ -0,0 +1,46 @@
+%define         debug_package %{nil}
+Summary:        Library implementing the jitter entropy source
+Name:           jitterentropy
+Version:        3.0.1
+Release:        1%{?dist}
+License:        BSD or GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Security
+URL:            https://github.com/smuellerDD/jitterentropy-library
+#Source0:       https://github.com/smuellerDD/%%{name}-library/archive/refs/tags/v%%{version}.tar.gz
+Source0:        %{name}-library-%{version}.tar.gz
+Patch0:         makefile.patch
+BuildRequires:  gcc
+BuildRequires:  make
+
+%description
+The Jitter RNG provides a noise source using the CPU execution timing jitter.
+It does not depend on any system resource other than a high-resolution time stamp.
+It is a small-scale, yet fast entropy source that is viable in almost all environments and on a lot of CPU architectures.
+
+
+%prep
+%autosetup -n %{name}-library-%{version} -p1
+
+%build
+%set_build_flags
+make
+
+%install
+mkdir -p %{buildroot}/usr/include/
+make install DESTDIR=%{buildroot} PREFIX=%{_prefix}
+
+# Rename static library to libjitterentropy-openssl
+mv %{buildroot}/%{_libdir}/lib%{name}.a %{buildroot}/%{_libdir}/lib%{name}-openssl.a
+
+%files
+%doc README.md
+%license COPYING COPYING.bsd COPYING.gplv2
+%{_includedir}/*
+%{_mandir}/man3/*
+%{_libdir}/lib%{name}-openssl.a
```

Comment:
Is it possible to build it as a shared library? Right now we'd need to update and re-release `openssl` whenever this package gets updated and we'll probably going to forget about it.

Reply:
This package can be built as a shared library, but Dan and I decided against this.

For FIPS certification, we want this library to be statically compiled with OpenSSL so the cryptographic boundary doesn't include shared libraries outside of OpenSSL's.

Reply:
I see. In that case do you have any process or mechanism in place to make sure we update all of the packages (currently only `openssl` I guess) building against this library as we update this package it in the future?